doc/android/AndroidNativeSamples.md

```diff
@@ -0,0 +1,67 @@
+#  Build RealSense SDK Samples for Android OS
+This document describes how to build the Intel® RealSense™ SDK 2.0 including headless tools and examples for Android devices.
+
+> Read about Android support [here](./Android.md).
+
+## Instructions
+1. [Root](https://www.wikihow.tech/Root-Android-Phones) your Android device.
+2. Download the [Native Development Kit (NDK)](https://developer.android.com/ndk/downloads/index.html) for Linux to your host machine.
+3. Install [CMake](https://cmake.org/) 3.6.1 or newer.
+4. Download [ADB](https://developer.android.com/studio/command-line/adb.html) to the host machine by typing `sudo apt-get install adb`.
```

Comment:
sudo apt-get install -qq appears in Travis but not here

Reply:
Because it's required only in Linux